### ISSUE TITLE
Built in Webconsole

### ISSUE THREAD

**Issue by alkhachatryan** [laravel/ideas#1491]
Hello.

I'd like to show you my small package, which will let people, who doesn't have an access to SSH to connect to remote server. It has a config file, view file, service provider. Let's test together and understand, do we need it in Laravel or not?

https://github.com/alkhachatryan/laravel-web-console

More information in Medium post
https://medium.com/@alexeykhachatryan/laravel-webconsole-or-i-use-a-shared-hosting-a1b49ef148ad

PS. I'm using unlimited hosting WITHOUT SSH connection and installed it.

**Comment by fletch3555** [comment]
I don't believe it should be in the core, since many hosts do have SSH enabled, and therefore not every Laravel user would need this functionality.  Best left as a package.

In either case, it's still a new package.  Let time prove it to be useful.

**Comment by alkhachatryan** [comment]
Thanks

**Comment by denjaland** [comment]
I agree it shouldn't be in the core, and I've just checked the package, but I don't understand: you're targetting audiences who do NOT have SSH access; how does this package work then?  For it t be able to execute commands on the server, it DOES need to have access, not?

**Comment by alkhachatryan** [comment]
> I agree it shouldn't be in the core, and I've just checked the package, but I don't understand: you're targetting audiences who do NOT have SSH access; how does this package work then? For it t be able to execute commands on the server, it DOES need to have access, not?

You cannot have a permission to run command from remote machine, but able to run from localhost (server).

I tested it on 4 servers from different hostings
